helpers.go

```diff
@@ -2,39 +2,23 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
-	"os"
 	"os/exec"
 	"strings"
 	"time"
 
 	log "github.com/sirupsen/logrus"
 )
 
-func getServiceRevisionFromIntegration(repo, baseBranch string) (string, error) {
-	tmp, _ := ioutil.TempDir("/var/tmp", "mender-integration")
-	defer os.RemoveAll(tmp)
-
-	gitcmd := exec.Command("git", "clone", "-b", baseBranch, "https://github.com/mendersoftware/integration.git", tmp)
-
-	// timeout and kill process after GIT_OPERATION_TIMEOUT seconds
-	t := time.AfterFunc(GIT_OPERATION_TIMEOUT*time.Second, func() {
-		gitcmd.Process.Kill()
-	})
-	defer t.Stop()
+func getServiceRevisionFromIntegration(repo, baseBranch string, conf *config) (string, error) {
+	c := exec.Command("release_tool.py", "--version-of", repo, "--in-integration-version", baseBranch)
```

Comment:
Note that you will still need to keep pulling the repository to keep it up to date. I'm not sure if that is handled now that this portion is gone.

Reply:
Yep. It is done once per PR here: https://github.com/mendersoftware/integration-test-runner/pull/29/files#diff-01127be9ed6b6352616a2f1c10bc3540R24